HybridManagement/HybridManagement.psd1

```diff
@@ -84,13 +84,14 @@
         "Get-AzStorageKerberosTicketStatus",
         "Test-AzStorageAccountADObjectPasswordIsKerbKey",
         "Update-AzStorageAccountADObjectPassword",
-        "Join-AzStorageAccountForAuth", 
+        "Join-AzStorageAccount", 

```

Comment:
Nit: If we're going to eventually have this cmdlet share a module with a bunch of other hybrid cmdlets, does "join-azstorageaccount" need something AD specific to the name.  I see we have an alias for the old name, do you think that's enough?

Reply:
We discussed this person, but for posterity, I believe this is what we should call this cmdlet: do the "join" action (verb) on the storage account object (noun). If we end up having other things that we need to "join" a storage account to, we can differentiate actions with parameter sets, although I struggle to imagine what else we might have.

I never liked the "ForAuth" part because it violates the PowerShell verb-noun construction. which is not to say we're perfectly doing this in other places of course.